sdk/python/setup.py

```diff
@@ -165,6 +165,17 @@
 else:
     use_scm_version = None
 
+# Build go server for 3 targets: macos (intel), macos (m1), linux (64 bit)
+# First start by clearing the necessary directory
+binaries_path = (pathlib.Path(__file__) / "../feast/binaries").resolve()
```

Comment:
We should ensure that this code only runs during dev/test environments and not when users pip install feast

Reply:
I've tried building the wheel using commands from release.yaml in the workflow and verified that binaries are shipped with the wheel